### PR TITLE
Make polyfill's test more readable

### PR DESCRIPTION
**Pull request by supl**
This PR revises the test files for polyfill.

We have two polyfills `bota` and `TextEncoder`.
They are implemented with pure JavaScript thus they are runnable on Node and Browsers.

There's no logical change.

They are identical to https://github.com/scalar-labs/scalardl-javascript-sdk-base/tree/master/polyfill

### CODE REVIEW

File: node/common/polyfill/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * from './btoa';
```
Comment (feeblefakie) [inline node/common/polyfill/index.ts]: Are these related to this PR?

**Review by jnmt — APPROVED** [review]
LGTM! Thank you!

File: node/common/polyfill/index.ts (hunk 1 above)
Comment (supl) [reply node/common/polyfill/index.ts]: Sorry.

This file should be added in another PR.

Allow me to remove it in https://github.com/scalar-labs/scalardl-javascript-client-sdk/pull/2/commits/2b8495413ab4448c24ca7c8ad58f60ccd139880e

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
